citadel-internal-service/tests/intra_kernel.rs

```diff
@@ -110,13 +114,15 @@ mod tests {
                 full_name: "Peer 0".to_string(),
                 username: "peer.0".to_string(),
                 password: "secret_0".to_string().into_bytes().to_owned(),
+                pre_shared_key: None::<PreSharedKey>,
             },
             RegisterAndConnectItems {
                 internal_service_addr: service_addr,
                 server_addr: server_bind_address,
                 full_name: "Peer 1".to_string(),
                 username: "peer.1".to_string(),
                 password: "secret_1".to_string().into_bytes().to_owned(),
+                pre_shared_key: None::<PreSharedKey>,
```

Comment:
```suggestion
                pre_shared_key: None,
```

On structs, you can define default types for when an optional is used.

E.g:

```rust
struct MyStruct<T: Into<String> = String> {
    value: Option<T>
}

let default_value = MyStruct { value: None }
// Note:  I don't have to specify "value: None::<String>" since I added the default type parameter to MyStruct
```

So, in the `RegisterAndConnectItems`' case, just default the `PreSharedKey` type to `PreSharedKey`.